### PR TITLE
Add Vue fixture.

### PR DESCRIPTION
**Pull request by jmhobbs**
Adds a trivial Vue file for #50

### CODE REVIEW

File: test/fixtures/code_samples/App.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+function setKms(e, v = +e.target.value) {
+  kms.value = v
+  miles.value = v / 1.609344
+}
+
+function setMiles(e, v = +e.target.value) {
+  miles.value = v
+  kms.value = v * 1.609344
+}
```
Comment (JoaoDanielRufino) [inline test/fixtures/code_samples/App.vue:31]: Could you add a single line comment and also a block comment anywhere here for tests purpose ?

**Review by JoaoDanielRufino — CHANGES_REQUESTED** [review]
Could you add the `vue` file to `scanner_test.go` ?

File: test/fixtures/code_samples/App.vue (hunk 1 above)
Comment (jmhobbs) [reply test/fixtures/code_samples/App.vue:33]: Sure thing, done!

**Comment by jmhobbs** [comment]
Added 👍

**Review by JoaoDanielRufino — APPROVED** [review]
Thanks for the amazing contribution!

**Pull request merged**